### PR TITLE
Closes #4618 - Stub Library UI tests 

### LINKED ISSUE #4618: Stub Library UI tests
See https://github.com/mozilla-mobile/fenix/issues/4572

### PR DESCRIPTION
**Pull request by AaronMT**
See #4572

### CODE REVIEW

**Comment by boek** [comment]
@rpappalax Could you give this a look? :)

File: app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    @Ignore("This is a stub test, ignore for now")
+    @Test
+    fun bookmarkTest() {
+        // Setup:
+        //  - Visit a URL
+        // Verify browser view exists
+        // Verify "slide-up tray" is visible
+        // Verify "Bookmark" is visible in "slide-up tray"
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: add step for saving bookmark

File: app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        // Verify "Bookmark saved" toast notification is visible
+        // Verify "Your Library" in 3-dot menu is visible
+        // Verify "Bookmarks" line-item in Library is visible
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: Add click on bookmarks

File: app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        // Verify "Bookmarks" UI elements
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: perhaps, Verify "Bookmarks" view is visible.

File: app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    @After
+    fun tearDown() {
+        mockWebServer.shutdown()
+    }
+
+    @Ignore("This is a stub test, ignore for now")
+    @Test
+    fun noHistoryInCacheTest() {
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt]: prob better as "historyItemsTest" for consistency

File: app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    @Ignore("This is a stub test, ignore for now")
+    @Test
+    fun historyTest() {
+        // Setup:
+        //  - Visit a URL
+        //  - Visit a second URL
+        // Verify browser view exists
+        // Verify "slide-up tray" is visible
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt]: (remove)

File: app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+        // Verify "History" UI elements
+        // Verify history is added, URLs match history added in Library
+
+        // Verify history 3-dot menu functions:
+        // 1. Edit
+        // 2. Copy
+        // 3. Share
+        // 4. Open in new tab
+        // 5. Open in private tab
+        // 6. Delete
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt]: remove items 1-5 (they don't currently exist), only Delete

File: app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+        // Verify history visibility in new URL search
```
Comment (rpappalax) [inline app/src/androidTest/java/org/mozilla/fenix/ui/HistoryTest.kt]: // add: return to 3 dot menu > Your Library

**Comment by rpappalax** [comment]
@AaronMT once these nits resolved, looks good 👍

**Pull request merged**